### ISSUE TITLE
Can you add Fleet please?

### ISSUE THREAD

**Issue by denofevil** [editorconfig/editorconfig.github.com#184]
Logo can be found here https://www.jetbrains.com/fleet/

**Comment by ffes** [comment]
Just wondering, isn't it better to add a general JetBrains reference instead of all their separate IDE products?

**Comment by xuhdev** [comment]
Yeah, @ffes's suggestions are worth considering. Maybe it still makes sense to add Fleet separately as it does seem to be materially different from the rest of JetBrain IDEs.

Would JetBrain be willing to contribute a stylished drawing of logo for Fleet?

**Comment by denofevil** [comment]
Yeah, I think we can link to https://www.jetbrains.com/ides/ then.
Let me figure out who can make a logo

**Comment by xuhdev** [comment]
Do you know any AI painter that stylizes an existing image? There should be some stylization websites but I couldn't find them.

**Comment by xuhdev** [comment]
I've asked for help on this one: https://softwarerecs.stackexchange.com/questions/90529/convert-images-to-paint-like-styles Wait to see what answer we can get

**Comment by verygoodsoftwarenotvirus** [comment]
Goland is additionally not referenced on the homepage, but does support EditorConfig

**Comment by xuhdev** [comment]
I'm a bit puzzled by the official Goland and Fleet logos. Could you point me out where I can download clear official copies of these two logos?

I found [Arbitrary Style Transfer in the Browser](https://reiinakano.com/arbitrary-image-stylization-tfjs/), which may be able to convert the style of the logos.

**Comment by denofevil** [comment]
Thanks a lot for looking into this!

They can be downloaded here: https://www.jetbrains.com/company/brand/#logos-and-icons
